test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -81,7 +79,7 @@ public void getControllerInfo() {
         List<URI> zkUris = zkService.getServiceDetails();
         log.info("zookeeper service details: {}", zkUris);
 
-        controllerService1 = Utils.createPravegaControllerService(zkUris.get(0));
+        controllerService1 = Utils.createPravegaControllerService("multicontroller", zkUris.get(0));
```

Comment:
Could we rename this variable as `controllerService`?

Reply:
done.